docs/content/latest/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -195,21 +206,94 @@ Tablet UUID                       Range                                 Leader
 a89ecb84ad1b488b893b6e7762a6ca2a  key_start: "\177\377" key_end: ""     127.0.0.3:9100
 ```
 
-{{< note title="Note" >}}
+{{< note title="Important" >}}
 
-There are a few important things to note here. 
-* The original tablet `9991368c4b85456988303cd65a3c6503` is no longer present in the system. It has been replaced with two new tablets. 
-* The tablet leaders are now spread across two nodes, in order to evenly balance the tablets for a table across the nodes in the cluster.
+* The original tablet `9991368c4b85456988303cd65a3c6503` no longer exists and has been replaced with two new tablets.
```

Comment:
Not sure how to mention this, but at least for now, that tablet still exists, until we enhance the code, to get rid of it. It's just that clients won't use it for reading/writing data.